### DISCUSSION TITLE
DNS integration & Azure VM support

### DISCUSSION THREAD

**Discussion by StuHare** [polymind-inc/acmebot#217] [Q&A] [answered]
Is it mandatory for the configuration to integrate with one of the DNS providers listed or is there an option where we can be provided with the DNS TXT record information, so that can be manually added to the DNS zone?

Is a new TXT record required and generated for each automated renewal?

Also does Acmebot support providing certs via the key vault to Azure VMs such as Windows and IIS?

Thanks

**Comment by shibayan** [comment, marked as answer]
>Is it mandatory for the configuration to integrate with one of the DNS providers listed or is there an option where we can be provided with the DNS TXT record information, so that can be manually added to the DNS zone?

Acmebot must be integrated with a DNS provider for full automation. Manual configuration is not possible.

>Is a new TXT record required and generated for each automated renewal?

Acmebot automatically creates and deletes the TXT records required for certificate renewal.

>Also does Acmebot support providing certs via the key vault to Azure VMs such as Windows and IIS?

There is no support for Azure VMs, but I think it can be done by preparing a script to get the certificate from Key Vault and set it up in the VM.

**Reply by StuHare** [reply]
Great thanks for the rapid response.
